src/main/scala/com/databricks/labs/overwatch/utils/Config.scala

```diff
@@ -14,7 +14,7 @@ class Config() {
   private final val packageVersion: String = getClass.getPackage.getImplementationVersion
   private val _isLocalTesting: Boolean = System.getenv("OVERWATCH") == "LOCAL"
   private var _debugFlag: Boolean = false
-  private var _overwatchSchemaVersion = "0.700"
+  private var _overwatchSchemaVersion = "0.701"
```

Comment:
The schema version should remain 0.700 as there is no schema update/upgrade between 0700 and 0701

Reply:
implemented